### DISCUSSION TITLE
Building and running the shapes demo

### DISCUSSION THREAD

**Discussion by PavloVlastos** [OpenDDS/OpenDDS#4522] [General]
My apologies if this is obvious and I just don't see it, but how do I build and run the shapes demo? I ran `configure` and `make` but do not see a shapes executable or similar in the examples folder.

**Comment by iguessthislldo** [comment]
I would recommend downloading our prebuilt binaries from the [releases page](https://github.com/OpenDDS/OpenDDS/releases) (under "Assets").

If you really need to build it, you should be able to do that with `configure --qt --tests` and `make ishapes` if the Qt5 development package is installed system-wide (`sudo apt install qtbase5-dev` on Debian/Ubuntu). If configure can't find Qt, then the [Qt document](https://github.com/OpenDDS/OpenDDS/blob/master/docs/qt.md) might help, but if not then feel free to ask.

**Comment by PavloVlastos** [comment]
Thanks! I ended up building form source - not for any particular reason. The arguments for the configure script worked. Thank you!
